Sources/QuickTests/Helpers/XCTestObservationCenter+QCKSuspendObservation.m

```diff
@@ -0,0 +1,23 @@
+@import XCTest;
+#import <objc/runtime.h>
+
+@interface XCTestObservationCenter (Redeclaration)
+- (NSMutableSet *)observers;
+@end
+
+@implementation XCTestObservationCenter (QCKSuspendObservation)
+
+static BOOL (^isFromApple)(id, BOOL *) = ^BOOL(id observer, BOOL *stop){
+    return [[NSBundle bundleForClass:[observer class]].bundleIdentifier containsString:@"com.apple.dt.XCTest"];
+};
+
+- (void)qck_suspendObservationForBlock:(void (^)(void))block {
+    NSSet *observersToSuspend = [[self observers] objectsPassingTest:isFromApple];
+    [[self observers] minusSet:observersToSuspend];
+
+    block();
+
+    [[self observers] unionSet:observersToSuspend];
```

Comment:
Not the `isFromApple` block, the `block` parameter—which is running the nested test suite. I figure any test suite that's being run here could possibly raise?